test/extended/testdata/bindata.go

```diff
@@ -12751,7 +12755,8 @@ var _examplesDbTemplatesMysqlEphemeralTemplateJson = []byte(`{
   },
   "message": "The following service(s) have been created in your project: ${DATABASE_SERVICE_NAME}.\n\n       Username: ${MYSQL_USER}\n       Password: ${MYSQL_PASSWORD}\n  Database Name: ${MYSQL_DATABASE}\n Connection URL: mysql://${DATABASE_SERVICE_NAME}:3306/\n\nFor more information about using this template, including OpenShift considerations, see https://github.com/sclorg/mysql-container/blob/master/5.7/README.md.",
   "labels": {
-    "template": "mysql-ephemeral-template"
+    "template": "mysql-ephemeral-template",
+    "app": "mysql-ephemeral-template"
```

Comment:
The `app` label should be parameterized so that when templates are used in conjunction to form a single application we can make the `app` labels match.